README.md

```diff
@@ -24,13 +24,15 @@ For a quick introduction to Joinmarket you can watch [this demonstration](https:
 * PayJoin - more economical and private payments between Joinmarket wallets.
 * Protection from [forced address reuse](https://en.bitcoin.it/wiki/Privacy#Forced_address_reuse) attacks.
 
-### Quickstart - RECOMMENDED INSTALLATION METHOD (Linux only)
+### Quickstart - RECOMMENDED INSTALLATION METHOD (Linux and MacOS only)
```

Comment:
Isn't correct spelling "macOS", not "MacOS"?

Reply:
Yes! Thanks for the review. I also corrected this elsewhere in the docs. See commits below.